core/src/components/item/item.ionic.scss

```diff
@@ -19,8 +19,10 @@
   --padding-end: #{globals.$ionic-space-400};
   --padding-bottom: #{globals.$ionic-space-200};
   --padding-start: #{globals.$ionic-space-400};
+  --color: #{globals.$ionic-color-neutral-1200};
```

Comment:
Since this color is close to black, the snapshots are going to look very similar.